src/openrct2-ui/windows/Tooltip.cpp

```diff
@@ -203,10 +203,10 @@ static void window_tooltip_paint(rct_window* w, rct_drawpixelinfo* dpi)
     gfx_filter_rect(dpi, left + 2, top + 0, right - 2, top + 0, PALETTE_DARKEN_3);
 
     // Corners
-    gfx_filter_pixel(dpi, left + 1, top + 1, PALETTE_DARKEN_3);
-    gfx_filter_pixel(dpi, right - 1, top + 1, PALETTE_DARKEN_3);
-    gfx_filter_pixel(dpi, left + 1, bottom - 1, PALETTE_DARKEN_3);
-    gfx_filter_pixel(dpi, right - 1, bottom - 1, PALETTE_DARKEN_3);
+    gfx_filter_pixel(dpi, ScreenCoordsXY{ left + 1, top + 1 }, PALETTE_DARKEN_3);
+    gfx_filter_pixel(dpi, ScreenCoordsXY{ right - 1, top + 1 }, PALETTE_DARKEN_3);
+    gfx_filter_pixel(dpi, ScreenCoordsXY{ left + 1, bottom - 1 }, PALETTE_DARKEN_3);
+    gfx_filter_pixel(dpi, ScreenCoordsXY{ right - 1, bottom - 1 }, PALETTE_DARKEN_3);
```

Comment:
```suggestion
    gfx_filter_pixel(dpi, { left + 1, top + 1 }, PALETTE_DARKEN_3);
    gfx_filter_pixel(dpi, { right - 1, top + 1 }, PALETTE_DARKEN_3);
    gfx_filter_pixel(dpi, { left + 1, bottom - 1 }, PALETTE_DARKEN_3);
    gfx_filter_pixel(dpi, { right - 1, bottom - 1 }, PALETTE_DARKEN_3);
```
Compiler can infer the type, so we usually do it this way for shortness

Reply:
Good to know! Let me make those changes.